staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go

```diff
@@ -154,7 +168,7 @@ func newOIDCAuthProvider(clusterAddress string, cfg map[string]string, persister
 	provider := &oidcAuthProvider{
 		client:    hc,
 		now:       time.Now,
-		cfg:       cfg,
+		cfg:       deepCopyStringMap(cfg),
```

Comment:
I need to think through what holding on to the original config means over the lifetime of a long running process.

Reply:
Any update on this? Are you OK with this approach?